.github/ISSUE_TEMPLATE/challenge_proposal.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 🚀 Challenge Proposal
+about: I want to add a new challenge!
+title: ''
+labels: challenge
+assignees: ''
+
+---
+
+## Describe the challenge you'd like to add
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+## Suggest a name
+The name of your baby. Consider that there might be adjustment in the challenge name.
+
+## Describe alternatives you've considered
```

Comment:
I read the title and then the description and I'm not sure how are they related 😅. Can you add some more info in the description 🙏